From a9260a2ae4bcf1385b787760e0329c0e48c7f7b9 Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@denx.de>
Date: Thu, 30 Jul 2020 19:30:27 +0200
Subject: [PATCH 6/6] ARM: stm32: Add fitImage its entry for 587-200 DHCOR SoM

The new 587-200 DHCOR SoM is compatible with the 587-100 prototype,
hence just replicate the entries, as there is no pattern matching.

Upstream-Status: Backport [b2a2911dadf028b5917e480312dc92f2e99e13d7]
Signed-off-by: Marek Vasut <marex@denx.de>
Cc: Patrice Chotard <patrice.chotard@st.com>
Cc: Patrick Delaunay <patrick.delaunay@st.com>
---
 board/dhelectronics/dh_stm32mp1/u-boot-dhcom.its | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/board/dhelectronics/dh_stm32mp1/u-boot-dhcom.its b/board/dhelectronics/dh_stm32mp1/u-boot-dhcom.its
index d8a1a28a11..905be57dff 100644
--- a/board/dhelectronics/dh_stm32mp1/u-boot-dhcom.its
+++ b/board/dhelectronics/dh_stm32mp1/u-boot-dhcom.its
@@ -44,12 +44,26 @@
 		};
 
 		config-2 {
+			/* DT+SoM+board model */
+			description = "dh,stm32mp15xx-dhcom-pdk2_somrev1_boardrev0";
+			loadables = "uboot";
+			fdt = "fdt-1";
+		};
+
+		config-3 {
 			/* DT+SoM+board model */
 			description = "dh,stm32mp15xx-dhcom-drc02_somrev0_boardrev0";
 			loadables = "uboot";
 			fdt = "fdt-2";
 		};
 
+		config-4 {
+			/* DT+SoM+board model */
+			description = "dh,stm32mp15xx-dhcom-drc02_somrev1_boardrev0";
+			loadables = "uboot";
+			fdt = "fdt-2";
+		};
+
 		/* Add 587-100..587-400 with fdt-2..fdt-4 here */
 	};
 };
-- 
2.27.0

